CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ Here is an overview of all new **experimental** features:
 
 - **General**: Centralize and improve automaxprocs configuration with proper structured logging ([#5970](https://github.com/kedacore/keda/issues/5970))
 - **General**: Paused ScaledObject count is reported correctly after operator restart ([#6321](https://github.com/kedacore/keda/issues/6321))
+- **General**: Revise some variable names and wrong test log([#6420](https://github.com/kedacore/keda/pull/6420))
```

Comment:
Let' move this into Other section - as it is not user facing fix. Or we can even remove it from the changelog. I am okay with both.

Reply:
> Let' move this into Other section - as it is not user facing fix. Or we can even remove it from the changelog. I am okay with both.

ok

Reply:
done, friendly ping @zroubalik @zroubalik , hope for your review, thank you.